app/main.py

```diff
@@ -1,7 +1,20 @@
-def get_weather() -> None:
-    # write your code here
-    pass
+import requests
+import os
+
+API_KEY = os.getenv("API_KEY")
+city = "Paris"
```

Comment:
According to the style guide, constants should be in uppercase. Consider renaming `city` to `CITY`.